src/components/ListCard.js

```diff
@@ -0,0 +1,24 @@
+//render a title for the List and ListItem components based on the props recieved from metaList
+
+import ListItem from "./ListItem";
+
+function ListCard() {
+  return (
+    <>
+    <ul>
```

Comment:
may use <div> instead of <ul> here because you have a ul and li inside this as well :)

Reply:
agree with Haeun here, unnecessary ul, prob could just remove it